packages/vertexai/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@firebase/vertexai",
-  "version": "0.0.1",
-  "description": "A template package for new firebase packages",
+  "name": "@firebase/vertexai-preview",
+  "version": "0.0.0",
+  "description": "A Firebase SDK for VertexAI",
```

Comment:
Suggestion: maybe we should we note that it's a preview version here in the description?

Reply:
Added, hopefully we catch this string when we go GA!